### PR TITLE
Fix XML code blocks 

### PR DESCRIPTION
**Pull request by cloudn9**
Continuing list from #9744

### CODE REVIEW

File: docs/visual-basic/misc/bc36558.md
Comment (gewarren) [inline docs/visual-basic/misc/bc36558.md]:
```suggestion
<AttributeUsage(AttributeTargets.All)>
```

Only the original example on line 31 was a run-on line.

**Review by gewarren — CHANGES_REQUESTED** [review]
One change to make.

File: docs/framework/data/adonet/connection-strings-and-configuration-files.md
Comment (Thraka) [inline docs/framework/data/adonet/connection-strings-and-configuration-files.md]: Missing end qoute
```suggestion
      type="System.Configuration.DpapiProtectedConfigurationProvider" />  
```

**Review by Thraka — APPROVED** [review]

**Comment by Thraka** [comment]
@gewarren Since you have a pending review request, I'll let you finish approval+merge. 👍

**Review by gewarren — APPROVED** [review]

**Comment by cloudn9** [comment]
Oh thank you both @gewarren @Thraka for correcting those.

**Pull request merged**
